bindings/protocols/bacnet/index.html

```diff
@@ -1,1192 +1,1751 @@
+            about the event that the current value is 55 and the exceeded limit was 50. The unit of these values can be found at the
+            <code>source</code> object that triggered this event.
+            Because overheating events are important for the operator to notice, <code>ackReq</code> is set, so the operator definitely
+            will see the event, even after the overheating condition is over.
+        </p>
+
+        <pre id="example-acknowledgeevent-input" class="example" title="Example AcknowledgeEvent input">
+            {
+                "source": "bacnet://5/0,20",
+                "toState": "offnormal",
+                "eventTime": "2024-07-30T12:33:00+00:00",
+                "ackSource": "Management Station Operator Acknowledge",
+                "ackTime": "2024-07-30T13:15:00+00:00"
+            }
+        </pre>
+
+        <p>
+            When the operator sees this event and decides to acnowledge it, management station will send the payload shown in
+            [[[#example-acknowledgeevent-input]]]. This can happen before or after the event is over.
+        </p>
+        
+        <pre id="example-getcurrentevents-output" class="example" title="Example GetCurrentEvents output">
+            {
+                "currentEvents": [
+                    {
+                        "source": "bacnet://5/0,12",
+                        "time": "2024-07-21T20:14:00+00:00",
+                        "prio": 200,
+                        "ackReq": 0,
+                        "toState": "fault"
+                    },
+                    {
+                        "source": "bacnet://5/0,20",
+                        "time": "2024-07-30T12:33:00+00:00",
+                        "prio": 150,
+                        "message": "Equipment overheat",
+                        "ackReq": 0,
+                        "toState": "offnormal",
+                        "values": {
+                            "deadband": 0.5,
+                            "exceedLim": 50,
+                            "exceedValue": 55,
+                            "statusFlags": [0, 0, 0, 0]
+                        }
+                    }
+                ],
+                "moreEvents": false
+            }
+        </pre>
+
+        <p>
+            If the event client wants to get a list of all events of the device, it will invoke the GetCurrentEvents action, and will receive
+            te payload shown in [[[#example-getcurrentevents-output]]]. Here the event notification about the overheating and a past fault event can be
+            seen. This is useful for synchronizing the events known on the client to the events that are on the device, in case of temporary
+            disconnection, reboot of client or device etc. Please note that in the second alarm <code>ackReq</code> value turned to 0,
+            because this event was acknowledged by the operator in the meantime.

```

Comment:
```suggestion
            If the event client wants to get a list of all events of the device, it will invoke the GetCurrentEvents action, and will receive
            te payload shown in [[[#example-getcurrentevents-output]]]. Here the event notification about the overheating and a past fault event can be
            seen. This is useful for synchronizing the events known on the client to the events that are on the device, in case of temporary
            disconnection, reboot of client or device, etc. Please note that in the second alarm, the <code>ackReq</code> value turned to 0,
            because this event was acknowledged by the operator in the meantime.
```